python/test/worker.config.json

```diff
@@ -3,6 +3,7 @@
         "language":"python",
         "extensions":[".py"],
         "defaultExecutablePath":"python",
-        "defaultWorkerPath":"worker.py"
+        "defaultWorkerPath":"worker.py",
+        "workerIndexing": "true"
```

Comment:
does this not affect legacy function tests?

Reply:
the second app setting prevents the host to enable/disable this feature.